masu/util/__init__.py

```diff
@@ -1 +1,51 @@
-# noqa
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Masu utility methods."""
+
+import os
+
+import watchtower
+from boto3.session import Session
+
+
+def setup_cloudwatch_logging(logger):
+    """Add Cloud Watch log handler if appropriate."""
+    key_id = os.environ.get('CW_AWS_ACCESS_KEY_ID')
+    secret = os.environ.get('CW_AWS_SECRET_ACCESS_KEY')
+    if not (key_id and secret):
+        logger.info('CloudWatch logging disabled due to missing access key')
+        return
+
+    session = Session(
```

Comment:
What happens to Masu if Cloudwatch isn't available?

Reply:
No logs get logged by that log handler into the AWS logging that is picked up by Kibana.

Reply:
There's no unit tests that show this and you've provided no functional test output to show that.

Can you please show some test results that demonstrate this code will continue functioning even when boto raises a `ClientError`?

Reply:
Yep, as I mentioned to Doug below. Clicked the PR button sooner than I meant to, while trying to multi-task in a meeting.